packages/two_dimensional_scrollables/lib/src/table_view/table.dart

```diff
@@ -803,7 +803,7 @@ class RenderTableViewport extends RenderTwoDimensionalViewport {
           axisDirectionIsReversed(horizontalAxisDirection)
               ? 0.0
               : _pinnedColumnsExtent,
-          axisDirectionIsReversed(horizontalAxisDirection)
+          axisDirectionIsReversed(verticalAxisDirection)
```

Comment:
Alas! A typo from https://github.com/flutter/packages/pull/5038 😅

Reply:
Ups, and I didn't catch it in my review 😝 

Thanks for adding a test for this case!